primitives/sr-primitives/src/traits.rs

```diff
@@ -713,6 +712,9 @@ pub trait SignedExtension: Codec + Debug + Sync + Send + Clone + Eq + PartialEq
 	/// The type that encodes information that can be passed from pre_dispatch to post-dispatch.
 	type Pre: Default;
 
+	/// An opaque set of information attached to the transaction.
```

Comment:
Should we add some examples to the description? Like what it could be used for or when it's not needed, etc.

Reply:
Did, though I must admit that this is not the end of the story, Once more of the refactors are done, we can technically use this to allow different runtimes, even different dispatches to carry different information in their `#[weight]`, but it would be super vague to formulate that + we will move stuff around soon again, so will probably change. I'll make a note for myself to elaborate on this after the repo is stable.